scst-store/using-encryption-and-connection.md

```diff
@@ -1,111 +1,79 @@
+The connection to the Store requires TLS encryption, the configuration depends on the kind of installation. Use the following instructions to set up the TLS connection, depending on which type of setup you are using:
 
-1. Using `LoadBalancer`
-1. Using `NodePort` — commonly used with local clusters such as kind, or minikube
+1. <a href="#ingress">Using `Ingress`</a>
+1. <a href="#no-ingress">Without `Ingress`</a>
+    - <a href="#use-lb">Using `LoadBalancer`</a>
+    - <a href="#use-np">Using `NodePort`</a> — commonly used with local clusters such as kind or minikube
 
-## <a id='use-lb'></a>Using `LoadBalancer`
 
-If you are using a `LoadBalancer` configuration, you need to find the external IP address of the `metadata-store-app` service. You can use kubectl to do this.
-
->**Note**: For all kubectl commands, use the `--namespace metadata-store` flag.
-
-```console
-kubectl get service/metadata-store-app --namespace metadata-store -o yaml
-```
-
-For example:
-
-```console
-$ kubectl get service/metadata-store-app --namespace metadata-store -o yaml
-...
-spec:
-  ports:
-  - name: http
-    nodePort: 32712
-    port: 8443
-    protocol: TCP
-    targetPort: PORT
-...
-status:
-  loadBalancer:
-    ingress:
-    - ip: IP-ADDRESS
-```
-
-Where:
+## <a id="ingress"></a>Using `Ingress`
 
-- `IP-ADDRESS` is your IP address.
-- `PORT` is your port number.
+When using an [Ingress setup](./ingress-multicluster.md), the Store creates a specific TLS Certificate for HTTPS communications under the `metadata-store` namespace.
 
-### <a id='obtain-ca-lb'></a>Obtaining the CA certificate
+To get such certificate you can use the following command:
 
-The CA certificate is generated by cert manager. Run the following command to get the CA certificate:
-
-```console
-kubectl get secret app-tls-cert -n metadata-store -o json | jq -r '.data."ca.crt"' | base64 -d > PATH
+```bash
+kubectl get secret ingress-cert -n metadata-store -o json | jq -r '.data."ca.crt"' | base64 -d > insight-ca.crt
 ```
 
-Where:
-
-- `PATH` is the location where you want to save the CA certificate. Use this file later when you configure the CLI. See [CLI plug-in configuration](../cli-plugins/insight/cli-configuration.md).
-
-If using an ingress, the CA certificate for clients is the certificate of the ingress domain:
+Additionally, the endpoint host will be set to `metadata-store.<ingress-domain>` (ex: `metadata-store.example.domain.com`), depending on the value of `ingress_domain` set during the setup.
 
-```console
-kubectl get secret ingress-cert -n metadata-store -o json | jq -r '.data."ca.crt"' | base64 -d > PATH
-```
+If no accessible DNS record exists for such domain, you can edit your `/etc/hosts` file to add a local record.
 
-### <a id='dns-resolution'></a>DNS resolution
+```bash
+ENVOY_IP=$(kubectl get svc envoy -n tanzu-system-ingress -o jsonpath="{.status.loadBalancer.ingress[0].ip}")
```

Comment:
Just out of curiosity, is `tanzu-system-ingress` the default namespace for contour when it's installed through TAP?

Reply:
yes, it is TAP's default contour installation